src/components/WebexSettings/WebexSettings.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {
+  Tab,
+  TabContent,
+  TabList,
+  TabPane,
+  Tabs,
+} from '@momentum-ui/react';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Webex Settings component
+ *
+ * @param {string} props.className Custom CSS class to apply
```

Comment:
- we should put a double space between the arg name and arg description
- better to also have `@param {object} props` at the top of the list